### PR TITLE
appflowy: 0.1.2 -> 0.1.3

### PR DESCRIPTION
**Pull request by DarkOnion0**
###### Description of changes

Update appflowy to the [latest version](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.1.3)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/office/appflowy/default.nix [hunk 1]
```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    cd AppFlowy/
+    cd AppFlowy_x86_64-unknown-linux-gnu_ubuntu-20.04/AppFlowy/
```
Comment (wegank) [inline pkgs/applications/office/appflowy/default.nix:42]: Looks like `stripRoot = false` can be removed?

Comment (DarkOnion0) [reply pkgs/applications/office/appflowy/default.nix:41]: After some testing, I was able to removed it, but not the ugly `cd` path

Comment (wegank) [reply pkgs/applications/office/appflowy/default.nix:41]: Did you update the hash?

Comment (DarkOnion0) [reply pkgs/applications/office/appflowy/default.nix:41]: I see, I will change this

Comment (DarkOnion0) [reply pkgs/applications/office/appflowy/default.nix:42]: Thanks for the reminder about changing the hash, but after changing it, I got the error message about multiple files/folders being in the archive root. To fix that, I added `stripRoot = false;` again.

Comment (wegank) [reply pkgs/applications/office/appflowy/default.nix:42]: Oh, that's unfortunate. Let's keep it as is.

**Review by wegank — APPROVED** [review]

**Pull request merged**